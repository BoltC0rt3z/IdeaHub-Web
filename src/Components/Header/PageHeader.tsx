import React from "react";
import { Row, Col, PageHeader as AntPageHeader, Avatar } from "antd";
import { AlignCenterOutlined, UserOutlined } from "@ant-design/icons";

import styles from "./PageHeader.module.scss";

const PageHeader: React.FC = () => (
  <Row className={styles.pageHeader}>
    <AntPageHeader
      title={
        <Row type="flex" gutter={16}>
          <Col>
            <AlignCenterOutlined className={styles.title} />
          </Col>
          <Col>Finite Analysis</Col>
        </Row>
      }
      extra={
        <Row>
          <Avatar
            icon={<UserOutlined />}
            style={{ background: "#001529", cursor: "pointer" }}
          />
        </Row>
      }
    />
  </Row>
);

export default PageHeader;
