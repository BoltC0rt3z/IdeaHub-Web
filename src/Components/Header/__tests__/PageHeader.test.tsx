import React from "react";
import { mount, ReactWrapper } from "enzyme";

import PageHeader from "../PageHeader";

describe("Tests for the page header", () => {
  const props = {
    openSideBar: jest.fn()
  };
  const wrapper: ReactWrapper = mount(<PageHeader {...props} />);

  it("Should test for the rendering of the page header without crushing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
