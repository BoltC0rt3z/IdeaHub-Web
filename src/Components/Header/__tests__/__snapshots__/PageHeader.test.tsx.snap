// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the page header Should test for the rendering of the page header without crushing 1`] = `ReactWrapper {}`;
